frontend/src/scenes/paths/NewPaths.js

```diff
@@ -62,18 +62,17 @@ function pageUrl(d, display) {
 
     let name = d.name.replace(/(^[0-9]+_)/, '')
 
+    if (!display) {
+        return name
+    }
+
     try {
         const url = new URL(name)
         name = incomingDomains.length !== 1 ? url.href.replace(/(^\w+:|^)\/\//, '') : url.pathname + url.search
```

Comment:
this was stripping too much from the URL (so returning the name before the URL hits this for functional purposes)

Reply:
Makes sense!